components/__tests__/Avatar.test.tsx

```diff
@@ -0,0 +1,55 @@
+// __tests__/Avatar.test.tsx
+
+import { render } from "@testing-library/react-native";
+import React from "react";
+
+import Avatar from "../Avatar"; // Adjust the path as needed
+
+// Mock the utility function
+jest.mock("@utils/getFirstLetterForAvatar", () => ({
```

Comment:
why do we have to mock this?

Reply:
ChatGPT thought it was a good idea, removed it!